Validation/EventGenerator/interface/GenWeightValidation.h

```diff
@@ -0,0 +1,72 @@
+#ifndef GENWEIGHTVALIDATION_H
+#define GENWEIGHTVALIDATION_H
+
+// framework & common header files
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
This interface is also not needed (perhaps there are a few more unused ones in this list of include's)